lib/yearbook/university.ex

```diff
@@ -295,4 +295,33 @@ defmodule Yearbook.University do
   def change_class(%Class{} = class, attrs \\ %{}) do
     Class.changeset(class, attrs)
   end
+
+  alias Yearbook.University.ClassStudent
+
+  def create_class_student(attrs \\ %{}) do
+    student = list_classes_student(attrs["student_id"], attrs["class_id"])
+
+    if student > 0 do
+      {:error, %Ecto.Changeset{}}
+    else
+      %ClassStudent{}
+      |> ClassStudent.changeset(attrs)
+      |> Repo.insert()
+    end
+  end
+
+  def list_classes_student(student_id, class_id) do
+    ClassStudent
+    |> where(student_id: ^student_id)
+    |> where(class_id: ^class_id)
+    |> Repo.all()
+    |> Enum.count()
+  end
```

Comment:
I don't think this is needed but I think it should be called student_classes_count :point_up: